### PR TITLE
docs: Update README with community and office hours info

### PR DESCRIPTION
**Pull request by joekr**
**What this PR does / why we need it**:
The readme should contain the information for our office hours and community engagement info

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
No open related issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,24 @@ The Cluster API itself is shared across multiple cloud providers allowing for tr
...
+
+- Chat with us on the Kubernetes [Slack](http://slack.k8s.io/) in the [#cluster-api-oci][#cluster-api-oci slack] channel
+
+### Office hours
+
+The community holds office hours on the first Tuesday of every month, with sessions open to all users and
+developers.
+
+Office hours are hosted on a [Zoom video chat][zoomMeeting] on the first Tuesday of every month
+at 06:00 PT / 09:00 ET / 14:00 UK / 18:00 India.
```
Comment (Djelibeybi) [inline README.md:42]:
```suggestion
The maintainers host office hours on the first Tuesday of every month
at 06:00 PT / 09:00 ET / 14:00 CET / 18:00 IST via [Zooom][zoomMeeting].

All interested community members are invited to join us. A recording of each session will be made available afterwards for folks who are unable to attend.
```

I was trying to make it a little more inclusive and a little less repetitive, but I'm not 100% happy with my rewrite.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,24 @@ The Cluster API itself is shared across multiple cloud providers allowing for tr
...
 
+## 🤗 Community
+
+The CAPOCI provider is developed in the open, and is constantly being improved by our users, contributors, and maintainers.
+
+If you have questions or want to get the latest project news, you can connect with us in the following ways:
+
+- Chat with us on the Kubernetes [Slack](http://slack.k8s.io/) in the [#cluster-api-oci][#cluster-api-oci slack] channel
+
+### Office hours
```
Comment (Djelibeybi) [inline README.md:36]:
```suggestion
### Office Hours
```

Just to be consistent with existing headings.

File: README.md [hunk 3]
```diff
@@ -25,6 +25,24 @@ The Cluster API itself is shared across multiple cloud providers allowing for tr
 
 You can find detailed documentation as well as a getting started guide in the [Cluster API Provider for OCI Book][capoci_book].
 
+## 🤗 Community
+
+The CAPOCI provider is developed in the open, and is constantly being improved by our users, contributors, and maintainers.
+
+If you have questions or want to get the latest project news, you can connect with us in the following ways:
+
+- Chat with us on the Kubernetes [Slack](http://slack.k8s.io/) in the [#cluster-api-oci][#cluster-api-oci slack] channel
```
Comment (Djelibeybi) [inline README.md:34]:
```suggestion
To ask questions or get the latest project news, please join us in the [#cluster-api-oci][#cluster-api-oci] channel on [Slack](http://slack.k8s.io/).
```

**Review by Djelibeybi — COMMENTED** [review]
My suggestions are just that: suggestions.

**Review by shyamradhakrishnan — APPROVED** [review]

**Pull request merged**